src/routes/tools/extreme-heat/_ExploreData.svelte

```diff
@@ -0,0 +1,460 @@
+  </div>
+
+  <div slot="stats">
+    <ul class="stats">
+      <li class="block">
+        <svelte:component
+          this="{$indicator.statsComponent}"
+          units="{$indicator.units}"
+          data="{statsData}"
+          isHistorical="{true}"
+          series="{'historical'}"
+          period="{'baseline'}"
+          format="{formatFn}"
+        />
+      </li>
+      <li class="block">
+        <svelte:component
+          this="{$indicator.statsComponent}"
+          units="{$indicator.units}"
+          data="{statsData}"
+          isHistorical="{false}"
+          series="{'future'}"
+          period="{'mid-century'}"
+          format="{formatFn}"
+        />
+      </li>
+      <li class="block">
+        <svelte:component
+          this="{$indicator.statsComponent}"
+          units="{$indicator.units}"
+          data="{statsData}"
+          isHistorical="{false}"
+          series="{'future'}"
+          period="{'end-century'}"
+          format="{formatFn}"
+        />
+      </li>
+    </ul>
+  </div>
+
+  <div slot="graphic" class="graphic block">
+    <svelte:component
+      this="{$indicator.chartComponent}"
+      data="{$data}"
+      dataByDate="{dataByDate}"
+      yAxis="{{
+        key: 'value',
+        label: `${$indicator.title}`,
+        baseValue: 0,
+        tickFormat: formatFn,
+        units: `${$indicator.units}`,
+      }}"
+    />
+    <div class="chart-notes margin--v-8">
+      <p>
+        Source: Cal-Adapt. Data: {$titles.join(", ")}.
+      </p>
+    </div>
+    <div class="chart-download margin--v-8">
+      <LearnMoreButton
+        cta="{'Explain Chart'}"
+        on:click="{() =>
+          loadLearnMore({
+            content: chartDescription,
+            header: 'About this Chart',
+          })}"
+      />
+      <div>
+        <Button size="small" icon="{Download16}" on:click="{loadDownload}">
+          Download Chart
+        </Button>
+        <Button size="small" icon="{Share16}" on:click="{loadShare}">
+          Share
+        </Button>
+      </div>
+    </div>
+  </div>
+
+  <div slot="settings" class="settings">
+    <div class="block">
+      <RadioBtnGroup
+        selected="{$climvarStore}"
+        items="{climvarList}"
+        title="Select Indicator"
+        on:change="{changeClimvar}"
+      />
+      <SelectClimvar
+        title=""
+        selectedId="{$indicatorStore}"
+        items="{indicatorList}"
+        on:change="{changeIndicator}"
+      />
+      <LearnMoreButton
+        on:click="{() =>
+          loadLearnMore({
+            slugs: ['extreme-heat-day', 'warm-night'],
+          })}"
+      />
+    </div>
+    <div class="block">
+      <SelectScenario
+        selectedId="{$scenarioStore}"
+        items="{DEFAULT_SCENARIOS}"
+        on:change="{changeScenario}"
+      />
+      <LearnMoreButton
+        on:click="{() => loadLearnMore({ slugs: ['emissions-scenario'] })}"
+      />
+    </div>
+    <div class="block">
+      <SelectThreshold
```

Comment:
I've removed the ability to remove custom thresholds as it's not in the current tool via https://github.com/berkeley-gif/caladapt-website-2021/pull/34/commits/907dd706e07ebe99fb64cff539501d939c0f18c5. This will simplify a lot of the user interaction. Also added 2 commonly used alternate values so the list folks can pick from is not just 1 radio button via https://github.com/berkeley-gif/caladapt-website-2021/pull/34/commits/57644c2547ef9e25ed64b7a28acbacfa36ab3691